packages/expo-keep-awake/ios/KeepAwakeModule.swift

```diff
@@ -45,7 +49,9 @@ public final class KeepAwakeModule: Module {
 }
 
 private func setActivated(_ activated: Bool) {
+  #if os(iOS) || os(tvOS)
   DispatchQueue.main.async {
     UIApplication.shared.isIdleTimerDisabled = activated
```

Comment:
rather than no-op, what do you think if we can show a console.warn for unimplemented feature?

Reply:
I'm not a big fun of warnings in that case. I think it's better to put this info in the documentation and let people just call it without checking if the feature is available.
So far our docs is not ready for a new platform, but I'm planning to work a bit on that. For now the goal is just to make it possible to add the expo package as the dependency to macOS projects.

Reply:
fair enough!
it's just that `KeepAwakeModule.swift` literally compiled but do nothing on macos. in the future we could find a better way to handle this unsupported platform code.
for example, not compiling KeepAwakeModule.swift for macos at all and do no-op on `ExpoKeepAwake.macos.ts`.